build/yamls/antrea-windows-with-ovs.yml

```diff
@@ -45,14 +62,12 @@ data:
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
     $env:PATH = $env:PATH + ";$mountPath/Windows/System32;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
-    $OVSDriverDir="$mountPath\openvswitch\driver"
 
-    # Check if OVSExt driver is already installed
-    $driverStatus = netcfg -q ovsext
-    if ($driverStatus -like '*not installed*') {
-      # Install OVS Driver
-      netcfg -l $OVSDriverDir/ovsext.inf -c s -i OVSExt
-    }
+    # Update Ovs Driver
```

Comment:
Why shall we copy the driver to Windows host? I remember you mentioned that Windows host can load the installed driver even if the file is not persistent on disk.

BTW, even if we have to copy the driver files, it should be taken in `Install-OVSDriver-Containerd.ps1` rather than in `Run-AntreaOVS-Containerd.ps1` right?

Reply:
The file path in the initcontainer cannot be shared with the OVS container. Therefore, after installing the driver in the initcontainer, we copy the driver file to the host to ensure it persists. Once the OVS container starts, it copies the driver from the host to its own file path. This ensures that the driver used is the same one installed in the initcontainer.